src/cic/model.clj

```diff
@@ -213,6 +205,56 @@
                 (let [next-placement (phase-transition (zero? offset) age placement)]
                   (recur next-offset next-placement (conj placements {:offset next-offset :placement next-placement})))))))))))
 
+(defn phase-durations
```

Comment:
I don't think we should do it right now, but once the model has stabilised with validation, I'd like to put the non-interacting bits of the model into separate namespaces under cic.model.*